src/cljs/main/broadfcui/utils/ajax.cljs

```diff
@@ -140,6 +140,21 @@
            :url (str (config/tos-url) "/v1/user/response" query)
            :headers (merge (@get-bearer-token-header) content-type=json)))))
 
+(defn call-metrics [event details]
+  (when (string? (config/metrics-url))
+    (try
+      (call {:method "POST"
+             :url (str (config/metrics-url) "/api/event")
+             :headers (merge (@get-bearer-token-header) content-type=json)
```

Comment:
should this be sending the bearer token? I don't know anything about the metrics service ....

Reply:
It should, our metrics service validates that users are registered before sending the anonymized event through.